web/handlers/auth/withssb.go

```diff
@@ -357,26 +366,42 @@ func (h WithSSBHandler) serverInitiated() (templateData, error) {
 	startAuthURI.Opaque = "experimental"
 	startAuthURI.RawQuery = queryParams.Encode()
 
-	// generate a QR code with the token inside so that you can open it easily in a supporting mobile app
-	qrCode, err := qrcode.New(startAuthURI.String(), qrcode.Medium)
-	if err != nil {
-		return templateData{}, err
-	}
+	var qrURI string
+	if isSolvingRemotely {
+		qrURI = ""
+	} else {
```

Comment:
```suggestion
	if !isSolvingRemotely {
```

`qrURI` doesnt need to be initiallized manually if we don't use it (empty string is the zero value for it anyhow).